nikola/post.py

```diff
@@ -277,12 +277,10 @@ def __repr__(self):
         return '<Post: {0!r} {1}>'.format(self.source_path, m.hexdigest())
 
     def _has_pretty_url(self, lang):
-        if self.pretty_urls and \
-                self.meta[lang].get('pretty_url', '') != 'False' and \
-                self.meta[lang]['slug'] != 'index':
-            return True
-        else:
-            return False
+        m = self.meta[lang].get('pretty_url', '')
+        return ((self.pretty_urls and m != 'False' and
+                 self.meta[lang]['slug'] != 'index') or
```

Comment:
It could be, but if you are specifying `pretty_url` as a special case, you probably don’t want it overridden by some magic. In fact, we could go the other way: allow `index` to create a pretty URL if there is an override, even if self.pretty_urls is True. Which way should I go?